### ISSUE TITLE
Exporting and Importing a Backup of your Decks

### ISSUE THREAD

**Issue by Wehzie** [cramkle/cramkle#585]
Exporting and importing would give some confidence that the Decks are still useful in case your server goes offline.
Pretty app.

**Comment by lucasecdb** [comment]
yeah, that's something I've been wanting to implement specially if you want to move to another app (such as Anki)
